cluster/images/controller-manager/Dockerfile

```diff
@@ -52,5 +52,8 @@ RUN go build -a -ldflags='-w -s -extldflags=static -X main.version=${VERSION}' -
 # Copy the manager into the distroless image.
 FROM ${DISTROLESS_IMAGE}
 LABEL maintainer="Travis Rhoden <trhoden@vmware.com>"
+
+USER nobody
```

Comment:
So this is the recommended non-root user for the distroless image?

Reply:
Yep! Other images based off distroless use it as well:
https://github.com/kubernetes/kube-state-metrics/blob/111c95ac200404aee140af9ff97c7f833e5dd087/Dockerfile